playbooks/10 - SP - FortiSOAR Health Assessment/FortiSOAR Health Assessment.json

```diff
@@ -0,0 +1,334 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "FortiSOAR Health Assessment",
+    "aliasName": null,
+    "tag": null,
+    "description": "",
+    "isActive": true,
+    "debug": true,
```

Comment:
@cs-rushikesh-mehakare  as we discussed, debug mode should be false for all the playbooks

Reply:
For QA let it be debug will change befour Final QA